pkg/funky/response.go

```diff
@@ -0,0 +1,33 @@
+package funky
+
+// Constants indicating the types of errors for Dispatch function invocation
+const (
+	InputError    = "InputError"
+	FunctionError = "FunctionError"
+	SystemError   = "SystemError"
+)
+
+// Response a struct to hold the response to a Dispatch function invocation
+type Response struct {
```

Comment:
This should really be named `Message`: both requests and responses are structured like this.